std/protocols/lookup.asm

```diff
@@ -100,4 +100,21 @@ let lookup: expr[], Fp2<expr>, Fp2<expr>, Constr, expr -> () = constr |acc, alph
     is_first * acc_1 = 0;
     is_first * acc_2 = 0;
     constrain_eq_ext(update_expr, from_base(0));
+
+    // In the extension field, we need a prover function for the accumulator.
+    if std::array::len(acc) > 1 {
+        // TODO: Helper columns, because we can't access the previous row in hints
+        let acc_next_col = std::array::map(acc, |_| std::prover::new_witness_col_at_stage("acc_next", 1));
```

Comment:
Can I give the same name for two columns?

Reply:
Or the prover function will make it unique?

Reply:
`acc_next_col` is not the name of the column, it's the name of an `expr` that references the column.

But generally, the system will ensure that names are unique and it will be an error if they are not.